<launch>
    <arg name="cmd_vel" default="/cmd_vel"/>
    <arg name="scan" default="/scan"/>

    <node pkg="obstacle_avoidance" type="simple_obstacle_stop_node" name="simple_obstacle_stop" output="screen">
        <param name="base_radius" type="double" value="0.2"/>
        <param name="slow_dis" type="double" value="1.0"/>
        <param name="stop_dis" type="double" value="0.5"/>
        <param name="linear_vel_up" type="double" value="0.5"/>
        <param name="linear_vel_low" type="double" value="0.05"/>
        <param name="slow_down_time" type="double" value="1"/>
        <param name="control_rate" type="int" value="20"/>

        <remap from="cmd_vel" to="$(arg cmd_vel)"/>
        <remap from="scan" to="$(arg scan)"/>
    </node>
</launch>